apps/ledger-live-desktop/src/renderer/hooks/useDeeplinking.js

```diff
@@ -43,10 +43,12 @@ export function useDeepLinkHandler() {
 
   const navigate = useCallback(
     (url: string, state?: any, search?: string) => {
-      if (url !== location.pathname) {
-        setTrackingSource("deeplink");
-        history.push({ pathname: url, state, search });
+      if (url === location.pathname && !search) {
```

Comment:
Currently, we are not navigating to the deep link if the pathname doesn't change. Therefore if we just change the query params, then the screen component never updates.

So I'm changing it to say that if the URL hasn't changed AND there's no search provided, then don't do the navigation... otherwise we can navigate.

Reply:
I've changed so it now reads as:

```
      if (url === location.pathname && search === location.search) {
        return;
      }
```

Hopefully this is a bit clearer with what's going on now?